.gitignore

```diff
@@ -3,3 +3,5 @@ lndr-server.config
 
 # Referenced in docker-compose.yml for persistence between sessions
 /.docker/
+
+ucac/node_modules
```

Comment:
I normally put /node_modules in my global .gitignore so it doesn't clutter every other .gitignore.